doc/requirements.txt

```diff
@@ -0,0 +1,3 @@
+myst-parser
+sphinx~=4.3.0
+breathe
```

Comment:
I think the following requirements would be helpful:
- ``sphinx-rtd-theme``: This would allow for local sphinx building during development
- `` sphinxcontrib-mermaid``: Used for mermaid diagrams in the docs

Reply:
Should be added now.